module/core/src/Infrastructure/Grid/UnitGrid.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Core\Infrastructure\Grid;
+
+use Ergonode\Core\Domain\ValueObject\Language;
+use Ergonode\Grid\AbstractGrid;
+use Ergonode\Grid\Column\LinkColumn;
+use Ergonode\Grid\Column\TextColumn;
+use Ergonode\Grid\Filter\TextFilter;
+use Ergonode\Grid\GridConfigurationInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ */
+class UnitGrid extends AbstractGrid
+{
+    /**
+     * @param GridConfigurationInterface $configuration
+     * @param Language                   $language
+     *
+     * @throws \Exception
+     */
+    public function init(GridConfigurationInterface $configuration, Language $language): void
+    {
+        $id = new TextColumn('id', 'Id', new TextFilter());
+        $id->setVisible(false);
+        $this->addColumn('id', $id);
+        $this->addColumn('name', new TextColumn('name', 'Unit name', new TextFilter()));
+        $this->addColumn('symbol', new TextColumn('symbol', 'Unit symbol', new TextFilter()));
+
+        $this->addColumn('_links', new LinkColumn('hal', [
+            'get' => [
+                'route' => 'ergonode_unit_read',
+                'parameters' => ['language' => $language->getCode(), 'unit' => '{id}'],
+            ],
+            'edit' => [
+                'route' => 'ergonode_unit_change',
+                'parameters' => ['language' => $language->getCode(), 'unit' => '{id}'],
+                'method' => Request::METHOD_PUT,
+            ],
+            'delete' => [
+                'route' => 'ergonode_unit_delete',
```

Comment:
Add proper privilege to all links
                'privilege' => '...._DELETE',